src/content/docs/data-apis/ingest-apis/metric-api/cardinality-management.mdx

```diff
@@ -52,19 +47,11 @@ In the **Metrics** table, you can see the breakdown of account cardinality by me
     src="/images/apis-and-data_screenshot-crop_metrics-table.webp"
 />
 
-This table shows the top 20 highest cardinality metrics in an account. The table provides the following details:
-* The name of the metric
-* The current overall usage of the metric (the ratio of current usage to current limit)
-* The actual values of the current cardinality
-* The current limit of that metric name 
-
-<Callout variant="tip">
-  All metrics have a default metric limit of 100,000.
-</Callout>
+All metrics have a default cardinality limit of 100,000. Based on your requirement, you can [adjust the per-metric limits](#per-metric-limit-adjustment).
 
 ## Metric cardinality breakdown [#metric-cardinality-breakdown]
 
-You can view a deeper layer of granularity in your metrics by clicking on any metric name. You can view a breakdown of the specific metric's cardinality details. 
+Select any metric in the <DNT>**Metrics**</DNT> table to view its detailed cardinality information in a dedicated <DNT>**Cardinality Management**</DNT> page.
 
 <img
```

Comment:
Justify this image

Reply:
The previous two images illustrated account cardinality, while this one focuses on metric cardinality. Since this is the first time we are describing cardinality, users who have not logged into New Relic One can use this image to become familiar with it.